admin/react/assets/src/dashboard-app/pages/fonts/edit/EditFont.js

```diff
@@ -0,0 +1,112 @@
+import { __ } from '@wordpress/i18n';
+import EditLocalFont from "./EditLocalFont";
+import EditGoogleFont from "./EditGoogleFont";
+import { RangeControl } from "@wordpress/components";
+import EditGooglePreviewItem from "./preview/EditGooglePreviewItem";
+import EditLocalPreviewItem from "./preview/EditLocalPreviewItem";
+
+const EditFont = ( props ) => {
+	const {
+		openPopup,
+		setOpenPopup,
+		font,
+		fontName,
+		fontType
+	} = props;
+
+	const [ open, setOpen ] = useState( openPopup );
+	const [ previewSize, setPreviewSize ] = useState( '20' );
+	const cancelButtonRef = useRef( null );
+
+	const onCancelClick = () => {
+		setOpenPopup( ! openPopup );
+	};
+
+	useEffect( () => {
+		setOpen( openPopup );
+	}, [ openPopup ] );
+
+	return (
+		<Transition.Root show={ open } as={ Fragment }>
+			<Dialog as="div" className="ast-edit-font__dialog fixed backdrop-blur-sm inset-0 overflow-y-auto" initialFocus={ cancelButtonRef } onClose={ setOpen }>
+				<div className="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0">
+					<Transition.Child
+						as={ Fragment }
+						enter="ease-out duration-300"
+						enterFrom="opacity-0"
+						enterTo="opacity-100"
+						leave="ease-in duration-200"
+						leaveFrom="opacity-100"
+						leaveTo="opacity-0"
+					>
+						<div className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
+					</Transition.Child>
+
+					{/* This element is to trick the browser into centering the modal contents. */}
+					<span className="hidden sm:inline-block sm:align-middle sm:h-screen" aria-hidden="true">
+						&#8203;
+					</span>
+					<Transition.Child
+						as={ Fragment }
+						enter="ease-out duration-300"
+						enterFrom="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+						enterTo="opacity-100 translate-y-0 sm:scale-100"
+						leave="ease-in duration-200"
+						leaveFrom="opacity-100 translate-y-0 sm:scale-100"
+						leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+					>
+						<div className="inline-block align-bottom bg-white rounded-lg text-left shadow-xl transform transition-all sm:my-8 sm:align-middle sm:max-w-[65%] sm:max-h-[50%] sm:w-full">
+							<div className="bcf-edit-modal-close-wrapper absolute right-[-15px] p-0 top-[-15px] w-[25px] h-[25px]">
+								<button
+									type="button"
+									className="mt-3 inline-flex justify-center border shadow-none border-slate-200 padding-[5px] bg-[#F6F7F7] text-base font-medium text-slate-800 focus:bg-[#F6F7F7] hover:bg-[#F6F7F7] focus:outline-none sm:mt-0 sm:text-xs bold border-none w-[20px] h-[20px] p-[3px] rounded-full shadow-md"
+									onClick={ onCancelClick }
+									ref={ cancelButtonRef }
+								>
+									<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" width="20" height="12" aria-hidden="true" focusable="false" className='mr-[2px]'><path d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"></path></svg>
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38